### PR TITLE
Censor /status and allow for custom status to be set (requires KFO Client update)

### PR DESCRIPTION
**Pull request by EstatoDeviato**

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
Can you name the PR descriptively, remember 1 PR per 1 core feature

File: server/area.py [hunk 1]
```diff
@@ -1720,9 +1720,21 @@ def change_status(self, value):
...
+            self.server.censors["whole"],
+            self.server.censors["replace"],
+            True,
+        )
+        value = censor(
+            value,
+            self.server.censors["partial"],
+            self.server.censors["replace"],
+            False,
+        )
```
Comment (Crystalwarrior) [inline server/area.py:1734]: why is this copy-pasted twice

File: server/area.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from server import commands
 from server.evidence import EvidenceList
 from server.exceptions import ClientError, AreaError, ArgumentError, ServerError
-from server.constants import MusicEffect, derelative
+from server.constants import MusicEffect, derelative, censor
```
Comment (Crystalwarrior) [inline server/area.py:5]: 👍

File: server/area.py [hunk 3]
```diff
@@ -1720,9 +1720,21 @@ def change_status(self, value):
...
+        value = censor(
+            value,
+            self.server.censors["partial"],
+            self.server.censors["replace"],
+            False,
+        )
+        if value.lower() == "hub":
             raise AreaError(
-                f'Invalid status. Possible values: {", ".join(allowed_values)}'
+                f'Invalid status. Possible values: {", ".join(possible_values)}'
```
Comment (Crystalwarrior) [inline server/area.py:1737]: Maybe just outward specify "hub" is a restricted value?

File: server/area_manager.py [hunk 4]
```diff
@@ -647,7 +647,7 @@ def send_arup_status(self, clients=None):
             return
         status_list = [1]
         if len(self.server.hub_manager.hubs) > 1:
-            status_list = [1, "GAMING"]
+            status_list = [1, "HUB"]
```
Comment (Crystalwarrior) [inline server/area_manager.py:650]: Pre-KFO Client might not display the right status color

**Review by Crystalwarrior — CHANGES_REQUESTED** [review]

File: server/area.py (hunk 1 above)
Comment (EstatoDeviato) [reply server/area.py:1734]: the first one censors["whole"] the second one is censors["partial"] I saw it in AO protocol file

File: server/area.py (hunk 3 above)
Comment (EstatoDeviato) [reply server/area.py:1737]: So just specify in ooc "Hub" is a restricted status?

File: server/area_manager.py (hunk 4 above)
Comment (EstatoDeviato) [reply server/area_manager.py:650]: What version KFO client will be? so I can diversify from the old one?

Comment (Crystalwarrior) [reply server/area_manager.py:650]: any client that isn't KFO LOL

**Pull request merged**